### PR TITLE
Closes #253 fix_bor_ne: Add CR and PR to bor_ne event

### LINKED ISSUE #253: Bug: Include CR and PR in bor_ne event
### What happened?

`"CR"` and `"PR"` should be included in the `bor_ne` event. Otherwise, subjects with a single `"PR"` or `"CR"` less than 42 days after randomization will have response `"MISSING"` instead of `"NE"` for confirmed best overall response.

### Session Information

_No response_

### Reproducible Example

_No response_

### PR DESCRIPTION
**Pull request by bundfussr**
Thank you for your Pull Request! We have developed this task checklist from the
[Development Process
Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process)
to help with the final steps of the process. Completing the below tasks helps to
ensure our reviewers can maximize their time on your code as well as making sure
the admiral codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task
or check off that it is not relevant to your Pull Request. This checklist is
part of the Github Action workflows and the Pull Request will not be merged into
the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update). Then the issue is closed automatically once it is merged to `main`.
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#writing-unit-tests-in-admiral)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiralonco/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

The following line requests to update the man pages by the "Man Pages" workflow.

/roxygenize

### CODE REVIEW

File: R/response_sources.R
Comment (rossfarrugia) [inline R/response_sources.R:137]: shouldn't this update be for cbor_ne rather than bor_ne? that's what we did for admiralroche.

File: NEWS.md
Comment (rossfarrugia) [inline NEWS.md:5]:
```suggestion
- The `bor_ne` event object was updated such that `CR` and `PR` are additionally
```

File: NEWS.md
Comment (rossfarrugia) [inline NEWS.md:7]:
```suggestion
patient with a single `CR` or `PR` assessment is derived as `"NE"` instead of
```

**Review by rossfarrugia — CHANGES_REQUESTED** [review]

File: NEWS.md
Comment (bundfussr) [reply NEWS.md]: Updated

File: NEWS.md
Comment (bundfussr) [reply NEWS.md]: Updated

File: R/response_sources.R
Comment (bundfussr) [reply R/response_sources.R:137]: We do not have `cbor_ne` in admiralonco. We use `bor_ne` for both BOR and CBOR.

Comment (bundfussr) [reply R/response_sources.R:137]: Do you think we should change this?

Comment (rossfarrugia) [reply R/response_sources.R:137]: ok, happy to go with your call as I see similar case with bor_sd. i'll approve for now and then if you decide to update i can re-review.

Comment (starosto) [reply R/response_sources.R:137]: we should also update ibor_ne for iRECIST

Comment (bundfussr) [reply R/response_sources.R:137]: Good point.
Updated

**Review by starosto — APPROVED** [review]

**Pull request merged**
